jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -471,6 +476,45 @@ private static Path getCheckedCacheDirectory(String property, Path defaultPath)
     return defaultPath;
   }
 
+  /**
+   * Checks whether or not the known default Java 8 distroless base image is being used. Checking
```

Comment:
_Checks if the base image uses a known Java 8 distroless image._ is shorter.  I wasn't sure what "default" meant, especially since Jib doesn't use the `:debug` images ever.

Reply:
Actually, I noticed this is a general method checking if the given image reference is a known distroless image. It doesn't have to be a base image. Reworded.